isthmus/src/test/java/io/substrait/isthmus/RoundingFunctionTest.java

```diff
@@ -0,0 +1,33 @@
+package io.substrait.isthmus;
+
+import java.util.List;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+public class RoundingFunctionTest extends PlanTestBase {
+
+  static List<String> CREATES =
+      List.of(
+          "CREATE TABLE numbers (i8 TINYINT, i16 SMALLINT, i32 INT, i64 BIGINT, fp32 FLOAT, fp64 DOUBLE, decimal DECIMAL(5,2)");
```

Comment:
Calcite doesn't like this CREATE statement because `decimal` is a keyword. Probably easiest to pick a different name.